### PR TITLE
OVS Test Data for Timestamp tests

### PR DESCRIPTION
**Pull request by kagemand-andersen**

### CODE REVIEW

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 1]
```diff
@@ -3,6 +3,117 @@
...
+INSERT INTO  dcsa_im_v3_0.party (
+    id,
+    party_name,
+    tax_reference_1,
+    tax_reference_2,
+    public_key,
+    address_id,
+    nmfta_code
+) VALUES ( 
+    uuid('be5bc290-7bac-48bb-a211-f3fa5a3ab3ae'),
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:38]:
```suggestion
    'be5bc290-7bac-48bb-a211-f3fa5a3ab3ae',
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 2]
```diff
@@ -3,6 +3,117 @@
...
+
+INSERT INTO dcsa_im_v3_0.location (
+    id,
+    location_name,
+    address_id,
+    latitude,
+    longitude,
+    un_location_code
+) VALUES (
+    uuid('06aca2f6-f1d0-48f8-ba46-9a3480adfd23'),
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:69]:
```suggestion
    '06aca2f6-f1d0-48f8-ba46-9a3480adfd23',
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 3]
```diff
@@ -73,18 +184,20 @@ INSERT INTO dcsa_im_v3_0.operations_event (
...
+    '8b64d20b-523b-4491-b2e5-32cfa5174eed',
     'ANA',
     'HAT',
-    'CRGO',
-    'this is definitely an event location',
-    'POTE'
+    'BUNK',
+    uuid('06aca2f6-f1d0-48f8-ba46-9a3480adfd23'),
+    'POTE',
+    uuid('be5bc290-7bac-48bb-a211-f3fa5a3ab3ae')
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:200]:
```suggestion
    'be5bc290-7bac-48bb-a211-f3fa5a3ab3ae'
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 4]
```diff
@@ -73,18 +184,20 @@ INSERT INTO dcsa_im_v3_0.operations_event (
...
     'DEPA',
-    uuid('8b64d20b-523b-4491-b2e5-32cfa5174eed'),
+    '8b64d20b-523b-4491-b2e5-32cfa5174eed',
     'ANA',
     'HAT',
-    'CRGO',
-    'this is definitely an event location',
-    'POTE'
+    'BUNK',
+    uuid('06aca2f6-f1d0-48f8-ba46-9a3480adfd23'),
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:198]:
```suggestion
    '06aca2f6-f1d0-48f8-ba46-9a3480adfd23',
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 5]
```diff
@@ -97,18 +210,20 @@ INSERT INTO dcsa_im_v3_0.operations_event (
...
-    uuid('123e4567-e89b-12d3-a456-426614174000'),
+    '123e4567-e89b-12d3-a456-426614174000',
     'ANA',
     'KAT',
     'WSDP',
-    'this is definitely also another event location',
-    'BRTH'
+    uuid('06aca2f6-f1d0-48f8-ba46-9a3480adfd23'),
+    'BRTH',
+    uuid('be5bc290-7bac-48bb-a211-f3fa5a3ab3ae')
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:226]:
```suggestion
    'be5bc290-7bac-48bb-a211-f3fa5a3ab3ae'
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 6]
```diff
@@ -97,18 +210,20 @@ INSERT INTO dcsa_im_v3_0.operations_event (
...
     TO_DATE('2003/05/03 21:02:44', 'yyyy/mm/dd hh24:mi:ss'),
     'ARRI',
-    uuid('123e4567-e89b-12d3-a456-426614174000'),
+    '123e4567-e89b-12d3-a456-426614174000',
     'ANA',
     'KAT',
     'WSDP',
-    'this is definitely also another event location',
-    'BRTH'
+    uuid('06aca2f6-f1d0-48f8-ba46-9a3480adfd23'),
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:224]:
```suggestion
    '06aca2f6-f1d0-48f8-ba46-9a3480adfd23',
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 7]
```diff
@@ -3,6 +3,117 @@
...
+);
+
+INSERT INTO dcsa_im_v3_0.vessel (
+    vessel_imo_number,
+    vessel_name,
+    vessel_flag,
+    vessel_call_sign_number,
+    vessel_operator_carrier_id
+) VALUES (
+    '1801323',
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:54]:
```suggestion
    '9321483',
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 8]
```diff
@@ -3,6 +3,117 @@
...
+    load_transport_call_id,
+    discharge_transport_call_id,
+    vessel_imo_number
+) VALUES (
+    'transport reference',
+    'Transport name (Singapore -> NYC)',
+    (SELECT mode_of_transport_code FROM dcsa_im_v3_0.mode_of_transport WHERE dcsa_transport_type = 'VESSEL'),
+    '7f2d833c-2c7f-4fc5-a71a-e510881da64a',
+    'b785317a-2340-4db7-8fb3-c8dfb1edfa60',
+    '1801323'
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:114]:
```suggestion
    '9321483'
```

**Review by HenrikHL — APPROVED** [review]

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 9]
```diff
@@ -3,6 +3,117 @@
...
+);
+
+INSERT INTO dcsa_im_v3_0.vessel (
+    vessel_imo_number,
+    vessel_name,
+    vessel_flag,
+    vessel_call_sign_number,
+    vessel_operator_carrier_id
+) VALUES (
+    '9321483',
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:54]:
```suggestion
    '9811000',
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 10]
```diff
@@ -3,6 +3,117 @@
...
+    load_transport_call_id,
+    discharge_transport_call_id,
+    vessel_imo_number
+) VALUES (
+    'transport reference',
+    'Transport name (Singapore -> NYC)',
+    (SELECT mode_of_transport_code FROM dcsa_im_v3_0.mode_of_transport WHERE dcsa_transport_type = 'VESSEL'),
+    '7f2d833c-2c7f-4fc5-a71a-e510881da64a',
+    'b785317a-2340-4db7-8fb3-c8dfb1edfa60',
+    '9321483'
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:114]:
```suggestion
    '9811000'
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 11]
```diff
@@ -3,6 +3,117 @@
...
+
+INSERT INTO dcsa_im_v3_0.vessel (
+    vessel_imo_number,
+    vessel_name,
+    vessel_flag,
+    vessel_call_sign_number,
+    vessel_operator_carrier_id
+) VALUES (
+    '9811000',
+    'King of the World',
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:55]:
```suggestion
    'Ever Given',
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 12]
```diff
@@ -3,6 +3,117 @@
...
+INSERT INTO dcsa_im_v3_0.vessel (
+    vessel_imo_number,
+    vessel_name,
+    vessel_flag,
+    vessel_call_sign_number,
+    vessel_operator_carrier_id
+) VALUES (
+    '9811000',
+    'Ever Given',
+    'DE',
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:56]:
```suggestion
    'PA',
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 13]
```diff
@@ -3,6 +3,117 @@
...
+    'NCVV',
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:57]:
```suggestion
    'H3RC',
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 14]
```diff
@@ -3,6 +3,117 @@
...
+    vessel_name,
+    vessel_flag,
+    vessel_call_sign_number,
+    vessel_operator_carrier_id
+) VALUES (
+    '9811000',
+    'Ever Given',
+    'PA',
+    'NCVV',
+    (SELECT id FROM dcsa_im_v3_0.carrier WHERE smdg_code = 'MSK')
```
Comment (HenrikHL) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:58]:
```suggestion
    (SELECT id FROM dcsa_im_v3_0.carrier WHERE smdg_code = 'EMC')
```

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 15]
```diff
@@ -73,18 +184,20 @@ INSERT INTO dcsa_im_v3_0.operations_event (
...
 ) VALUES (
-    uuid('a7a30556-ef7c-11eb-9a03-0242ac130003'),
+    uuid('b785317a-2340-4db7-8fb3-c8dfb1edfa60'),
     'ACT',
     TO_DATE('2003/05/03 21:02:44', 'yyyy/mm/dd hh24:mi:ss'),
     'DEPA',
-    uuid('8b64d20b-523b-4491-b2e5-32cfa5174eed'),
+    '8b64d20b-523b-4491-b2e5-32cfa5174eed',
     'ANA',
     'HAT',
```
Comment (Abdiiir) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:196]: PublisherRole is an ENUM and does not inlude "HAT"

File: datamodel/testdata.d/08_03_test_data_ovs.sql [hunk 16]
```diff
@@ -97,20 +210,22 @@ INSERT INTO dcsa_im_v3_0.operations_event (
...
+    publisher
 ) VALUES (
     uuid('03482296-ef9c-11eb-9a03-0242ac130003'),
     'EST',
     TO_DATE('2003/05/03 21:02:44', 'yyyy/mm/dd hh24:mi:ss'),
     'ARRI',
-    uuid('123e4567-e89b-12d3-a456-426614174000'),
+    '123e4567-e89b-12d3-a456-426614174000',
     'ANA',
     'KAT',
```
Comment (Abdiiir) [inline datamodel/testdata.d/08_03_test_data_ovs.sql:222]: PublisherRole is an ENUM and does not inlude "KAT"

**Review by Abdiiir — COMMENTED** [review]
Refer to SwaggerHub for PublisherRole Enums. 
Otherwise, it LGMT

**Comment by HenrikHL** [comment]
Good catch @Abdiiir - I have made a new PR fixing this... #128

**Pull request merged**